_opensearch/install/upgrade-overview.md

```diff
@@ -0,0 +1,72 @@
+---
+layout: default
+title: Upgrade Overview
+parent: Install OpenSearch
+nav_order: 999
+---
+
+# Upgrade Overview
+
+The OpenSearch Project releases regular updates that include new features, enhancements, and bug fixes. OpenSearch uses [Semantic Versioning](https://semver.org/), which means that breaking changes are only introduced between major version releases. To learn about upcoming features and fixes, review the [OpenSearch Project Roadmap](https://github.com/orgs/opensearch-project/projects/1) on GitHub. To see a list of previous releases, or to learn more about how OpenSearch uses versioning, check out the [Release Schedule and Maintenance Policy]({{site.url}}/releases.html).
+
+OpenSearch nodes and indexes are backwards-compatible by one major version. That means, for example, that you can restore an index to an OpenSearch 2.4.1 cluster from a snapshot that was taken on an OpenSearch 1.x cluster. Index compatibility is determined by the version of OpenSearch or Lucene that created the index. If an index was created by an OpenSearch cluster running version 1.0, then the index could be used by any other OpenSearch cluster running up to the latest 1.x or 2.x release.
+
+If your upgrade path spans more than a single major version, then you will need to use the [Reindex]({{site.url}}{{site.baseurl}}/api-reference/document-apis/reindex/) API to make your indexes compatible with the target version of OpenSearch. For example, if your cluster is currently running Elasticsearch 6.8 and you want to upgrade to OpenSearch 2.x, then you must first upgrade to OpenSearch 1.x, recreate your indexes using the [Reindex]({{site.url}}{{site.baseurl}}/api-reference/document-apis/reindex/) API, and finally upgrade to 2.x.
+
+Reindexing data is a resource-intensive process. We recommend that you only reindex when necessary.
```

Comment:
I was originally thinking it would be good to suggest a reindex as part of the change management process to make sure that any static indices that don't get rotated out won't break unexpectedly.

It just seems like tracking Lucene versions of existing indices is tedious but maybe I'm wrong.